draft-ietf-mls-protocol.md

```diff
@@ -831,12 +844,14 @@ other than for computing other secrets.)
 
 Clients MUST validate remote public values by ensuring
 that the point is a valid point on the elliptic curve.
-The appropriate validation procedures are defined in Section 4.3.7 of {{X962}}
-and alternatively in Section 5.6.2.3 of {{keyagreement}}.
-This process consists of three steps: (1) verify that the value is not the point at
-infinity (O), (2) verify that for Y = (x, y) both integers are in the correct
-interval, (3) ensure that (x, y) is a correct solution to the elliptic curve equation.
-For these curves, implementers do not need to verify membership in the correct subgroup.
+The appropriate validation procedures are defined in Section 4.3.7
+of {{X962}} and alternatively in Section 5.6.2.3 of {{keyagreement}}.
+This process consists of three steps: (1) verify that the value is not
+the point at infinity (O), (2) verify that for Y = (x, y) both integers
+are in the correct interval, (3) ensure that (x, y) is a correct solution
+to the elliptic curve equation.
+For these curves, implementers do not need to verify membership in
```

Comment:
Nit: You could probably move this up to the previous line.   Since there's not a blank line, it's not a new paragraph.

Reply:
Done